compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -272,6 +281,20 @@ FailureOr<PadPackTiling> PadPackTiling::create(linalg::LinalgOp linalgOp) {
   uint32_t M0 = findLargestFactor(M, maxL0Size, M1);
   uint32_t N0 = findLargestFactor(N, maxL0Size, N1);
 
+  auto downScale = [](uint32_t pack, uint64_t memTile, uint64_t coreTile) {
```

Comment:
maybe a comment saying this is doing step 3 or some summary of what step 3 above does.